ovirt/resource_ovirt_vm_test.go

```diff
@@ -596,3 +597,21 @@ resource "ovirt_vm" "vm" {
 }
 `, clusterID, templateID)
 }
+
+type OSTestParams struct {
+	osType   string
+	expected bool
+}
+
+var osCases = []OSTestParams{
+	{
+		"rhcos_x86",
+		true,
+	},
+}
+
+func TestVMOperatingSystemExpansion(t *testing.T) {
+	os, err := expandOS([]interface{}{osCases})
```

Comment:
This makes CI failed with message `ovirt/resource_ovirt_vm_test.go:614:35: cannot use []interface {} literal (type []interface {}) as type *schema.ResourceData in argument to expandOS`. Could you please fix this? Thanks.

Reply:
done